### ISSUE TITLE
Can this be used for ubuntu USB drive installations?

### ISSUE THREAD

**Issue by nrmnrm** [yusefnapora/pixelbook-linux#1]
First of all lots of "thank yous" for this impressive guide and the work that went into it!

As I am not ready to say good bye to chromeos, I already have a an usb drive ubuntu installation that works fairly ok. Seeing you made audio work makes me want to try your script (so just the stuff without the firmware flashing and WP disabling).
Just wanted to make sure your script does only work on the root partitions it is installed into, so it is not doing anything harmful to my chromeos partitions!? 

As a sidenote: any idea why the uefi firmware has to break the abilty to start chromeos?

Thanks again for the great work and helping me out :)

**Comment by flantel** [comment]
Looking at the Ansible code I don't see why not. I will give that a shot first, creating the usb drive and running this ansible install on a totally separate machine, then see how it works. I'll report back.

**Comment by flantel** [comment]
Sorry, I went ahead and installed directly. Seeing as I had to switch to Dev mode anyway I decided to go the whole hog.
Impressed!

**Comment by yusefnapora** [comment]
Sorry I haven't had time to respond earlier @nrmnrm - I think it should be safe, but if you look at #3 it seems that someone using a dual-boot setup hit a snag with extracting the recovery files. I'm not sure whether that would affect a USB install, but it seems like it might.

If you give it a shot, let me know how it works out - I might have some time this weekend to play with it also, since booting from USB is a bit easier for me to test than the dual-boot setup in the other issue.

**Comment by nrmnrm** [comment]
So I dared to run the script, getting slightly different results with my usb stick install than https://github.com/yusefnapora/pixelbook-linux/issues/3

Mine is:
TASK [eve-recovery-files : Find mapper device to use for mounting image] ***********************
fatal: [localhost]: FAILED! => {"msg": "Unexpected templating type error occurred on ({{ kpartx_output.stdout | regex_search(regexp, '\\\\1') | first }}): 'NoneType' object is not iterable"}
	to retry, use: --limit @/home/nrm/improve-ubuntu/pixelbook-linux/ansible/playbook.retry

PLAY RECAP *************************************************************************************
localhost                  : ok=9    changed=5    unreachable=0    failed=1   

Any ideas?

**Comment by nrmnrm** [comment]
happy to report back the fix suggested here:
https://github.com/yusefnapora/pixelbook-linux/pull/4
makes everything work already. 
only issue left: screen is rotated in the beginning, guess this is because of tablet mode?!
needed to execute xrandr -o normal

this is perfect work, screen dimming and sound working out of the box!

**Comment by flantel** [comment]
@nrmnrm so are you running this off USB successfully now?

**Comment by nrmnrm** [comment]
yes, works really well. Still wondering why login screen is rotating, guess i have to check iio-sensor-proxy

**Comment by nrmnrm** [comment]
I just had to deactivate rotating in login and regular screen (using ubuntu on-board tools in top right menu), now everything is just great!
